packages/account/imAccount/validator/P256Owner.ts

```diff
@@ -0,0 +1,95 @@
+import { p256 } from "@noble/curves/p256"
+import * as ethers from "ethers"
+
+import byte from "~packages/util/byte"
+import type { BytesLike, HexString } from "~typing"
+
+import type { WebAuthnValidatorOwner } from "../validator"
+
+export class P256Owner implements WebAuthnValidatorOwner {
+  public credentialId: string
+  public privateKey: Uint8Array
+  public publicKey: Uint8Array
+  public x: bigint
+  public y: bigint
+
+  public defaultRpidHash =
+    "0x49960de5880e8c687434170f6476605b8fe4aeb9a28632c7995cf3ba831d9763"
+  public defaultFlagsInt = "0x05"
+  public defaultSignCount = "0x00000002"
+
+  public constructor() {
+    this.privateKey = p256.utils.randomPrivateKey()
+    this.publicKey = p256.getPublicKey(this.privateKey)
+
+    const point = p256.ProjectivePoint.fromPrivateKey(this.privateKey)
+    this.x = point.x
+    this.y = point.y
+  }
+
+  public use(credentialId: string) {
+    this.credentialId = credentialId
+  }
+
+  public async sign(challenge: BytesLike) {
+    if (!this.credentialId) {
+      throw new Error("Credential id is not set")
+    }
+    const clientDataJson = {
+      type: "webauthn.get",
+      challenge: byte.normalize(challenge).toString("base64url"),
+      origin: "http://localhost:5173",
+      crossOrigin: false
+    }
+    const clientData = JSON.stringify(clientDataJson)
```

Comment:
```suggestion
    const clientData = JSON.stringify(clientDataJson, [
      "type",
      "challenge",
      "origin",
      "crossOrigin"
    ])
```
Because clientDataJSON follows [specific serialization rules](https://www.w3.org/TR/webauthn-3/#clientdatajson-serialization), it is recommended to specify the order of stringify here.